specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkManagerGroupMembership.json

```diff
@@ -0,0 +1,252 @@
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{virtualNetworkName}/providers/Microsoft.Network/networkGroupMemberships": {
+      "parameters": [
+        {
+          "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+        },
+        {
+          "$ref": "../../../../../common-types/resource-management/v4/types.json#/parameters/ResourceGroupNameParameter"
+        },
+        {
+          "$ref": "#/parameters/VirtualNetworkNameParameter"
+        },
+        {
+          "$ref": "./network.json#/parameters/ApiVersionParameter"
+        },
+        {
+          "$ref": "#/parameters/FirstIndexParameter"
+        },
+        {
+          "$ref": "#/parameters/PageSizeParameter"
+        }
+      ],
+      "get": {
+        "tags": [
+          "NetworkGroupMemberships_List"
+        ],
+        "operationId": "ListNetworkGroupMemberships",
+        "description": "List networkGroupMemberships applied on a virtual network.",
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the configurations.",
+            "schema": {
+              "$ref": "#/definitions/NetworkGroupMembershipsListResult"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List network group memberships": {
+            "$ref": "./examples/NetworkGroupMembershipsList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkGroupMembershipsListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkGroupMembership"
+          },
+          "description": "Gets a page of NetworkGroupMembership"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets the URL to get the next page of results."
+        }
+      },
+      "description": "Result of the request to list NetworkGroupMembership."
+    },
+    "NetworkGroupMembership": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkGroupMembershipProperties",
+          "description": "The network group membership properties"
+        },
+        "location": {
```

Comment:
is this the location of the virtual network? Maybe update the description to be more specific if it is

Reply:
also, why did we make the call to have location outside of the properties?

Reply:
Yes, it is for vnet location. And for all resources, location is out of properties. Please check.